internal/sbi/router.go

```diff
@@ -45,6 +45,10 @@ func newRouter(s *Server) *gin.Engine {
 	spyFamilyGroup := router.Group("/spyfamily")
 	applyRoutes(spyFamilyGroup, s.getSpyFamilyRoute())
 
+	saoGroup := router.Group("/sao")
+	applyRoutes(saoGroup, s.getSAORoute())
+	applyRoutes(saoGroup, s.postSAORoute())
```

Comment:
combin to `getSAORoute()`.
The get here is just get, not "GET" in http method. 
You can refer to each NF